projects/cdk/abstract/controller.ts

```diff
@@ -1,6 +1,8 @@
 import {OnChanges} from '@angular/core';
 import {Subject} from 'rxjs';
 
+// TODO: Class name `TuiController` must have one of the following prefixes: Abstract, TuiAbstract
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export abstract class TuiController implements OnChanges {
```

Comment:
Lets mark it as deprecated.
Then create new selector
```ts
export abstract class AbstractTuiController extends TuiController {}
```

\+ write migration (or add task to `https://github.com/Tinkoff/taiga-ui/issues/1764`)

Reply:
fixed, and add task to https://github.com/Tinkoff/taiga-ui/issues/1764